Individual.cpp

```diff
@@ -216,5 +217,31 @@ ostream& operator<<(ostream &fout, Individual& ind)
 	return fout;
 }
 
+void Individual::setIndividualMatchFile()
+{
+	string ext = ".tsv";
+	string dir = "individual_match_files/";
+	string fileHandleName = dir + single_id + ext;
+	individualMatchFile = new ofstream(fileHandleName, ofstream::app);
+}
+
+void Individual::setIndividualHomozFile()
+{
+	string ext = ".tsv";
+	string dir = "individual_homoz_files/";
+	string fileHandleName = dir + single_id + ext;
```

Comment:
Rust code names file like:

`{}/{}/chr{:02}.tsv`
first parameter is `dog_level_homoz_files`
Second parameter is new_dog_genotype_proxy_key
And then chromosome is chromosome

We aren't passing chromosome to germline right now, so maybe we need to pass it in? I don't think the data loader classes nor the match class contain data about chromosome?

Reply:
Oh I guess chromosome is loaded in.. but still creates a challenge because I don't think the program is written with the expectation it will be run on just 1 chromosome (so the chromosome isn't stored high up anywhere

Reply:
Added it as executable parameter, use to generate the filepaths